src/components/ProfitAndLossSummaries/ProfitAndLossSummaries.tsx

```diff
@@ -54,149 +56,113 @@ const buildMiniChartData = (scope: Scope, data?: ProfitAndLoss) => {
   return items.slice()
 }
 
+const SECTION_CLASS_NAME = 'Layer__ProfitAndLossSummaries'
+const SECTION_CLASS_NAMES = `${SECTION_CLASS_NAME} Layer__component`
```

Comment:
We do have existing terms `View` and `Component` - is your concept of `Section` independent from those? I _think_ it would be based on your description that sections can go inside of other sections: sections are just whatever gets publicly exposed, which could be components or views (or something in the future).

Reply:
`Component` seems like a wildcard term - it has no indication of it's role in a hierarchy.

Section has an semantic meaning in html (`<section>...</section>`). Sections are supposed to be siblings of each other, and they can be placed within a layout.

It seems like our `View` is a layout that has been populated with several sections. A `View` is the faster way to get something working with our components, but should not - in my opinion - support every opinion of the wrapped sections.